web_ckeditor4/models/ckeditor_monkeypatch.py

```diff
@@ -27,7 +27,7 @@ class CkeditorMonkeypatch(models.AbstractModel):
     _name = 'ckeditor.monkeypatch'
     _description = 'Monkeypatches for CKEditor'
 
-    def _register_hook(self, cr):
+    def _register_hook(self):
         marker = self._name.replace('.', '_')
         if not hasattr(clean, marker):
             # monkey patch lxml's html cleaner to allow image data urls
```

Comment:
@daramousk this one was necessary because lxml cleaned away all dataurls, but meanwhile, [my patch](https://github.com/lxml/lxml/pull/162) for this is merged in lxml, so I don't think we need this in the new version

Reply:
@hbrunn I see that your commit exists from lxml 3.6 onwards while v10 has version 3.5.0 pinned. 

git branch  -a --contains 763a7c4d05a6a9031e22488d834600610ce28762
* master
  remotes/origin/HEAD -> origin/master
  remotes/origin/LP1595781_adopt_external_doc
  remotes/origin/lxml-3.6
  remotes/origin/lxml-3.7
  remotes/origin/master
  remotes/origin/pypy4

https://github.com/OCA/OCB/blob/10.0/requirements.txt#L10

Would it still be correct to change it?

Reply:
no. Thanks for checking!